api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ExceptionControllerAdvice.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Through to limitations in the git compare mode, it's hard to review this class here.

The new codes expands existing methods to enormous sizes. I don't like that as it's untestable.
Create methods, preferly in corelib so it can be reused more easily.

in principle Controller classes should not contain any business logic at all.
If you create helper methods, first ask yourself if they should be in this controller, and then if they are public or private. If they are only used in this controller, make them private and place them at the bottom of the class. If they are used by other controllers, they DO NOT belong in this class.